packages/react-scripts/template/README.md

```diff
@@ -398,15 +398,14 @@ Following this rule often makes CSS preprocessors less useful, as features like
 First, let’s install the command-line interface for Sass:
 
 ```
-npm install node-sass --save-dev
+npm install node-sass-chokidar --save-dev
 ```
-
 Then in `package.json`, add the following lines to `scripts`:
 
 ```diff
    "scripts": {
-+    "build-css": "node-sass src/ -o src/",
-+    "watch-css": "npm run build-css && node-sass src/ -o src/ --watch --recursive",
```

Comment:
Why do we build before we start watching? Does the watcher not compile everything when started in watch mode?